### PR TITLE
Add a bio for @kphutt

### PR DESCRIPTION
**Pull request by kphutt**
This Pull Request adds a bio for @kphutt as requested in #23.

@JoshuaSCochrane what do you think of these changes? I would :heart: your :eyes: and :thought_balloon: on this.

### CODE REVIEW

**Comment by kphutt** [comment]
These are the _best_ changes ever. Perfect as is.

File: docs/kphutt.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# KPHutt
+
+**Name:** Karsten Huttelmaier
+
+**Occupation:** Software Engineer
+
+**Hometown:**  Mequon, WI
```
Comment (kphutt) [inline docs/kphutt.md:7]: Is this really where you live?

:earth_asia: 
:earth_africa: 
:earth_americas:

File: docs/kphutt.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# KPHutt
+
+**Name:** Karsten Huttelmaier
```
Comment (dmellenthin) [inline docs/kphutt.md]: Please verify the spelling

Comment (kphutt) [inline docs/kphutt.md]: I made a typo, the actual spelling should be Carlston Hootmayer. Good catch!

**Comment by kphutt** [comment]
Regarding the comment - [here](https://github.com/githubteacher/developers-09-24-2015/pull/36#discussion_r40336237)

**Comment by dmellenthin** [comment]
This comment(https://github.com/githubteacher/developers-09-24-2015/pull/36#discussion_r40336237)

**Comment by kphutt** [comment]
This is getting meta https://github.com/githubteacher/developers-09-24-2015/pull/36#issuecomment-142973383

**Comment by kphutt** [comment]
Hmm that didn't work out. This is becoming a chat room.

**Comment by kphutt** [comment]
:+1: 

:shipit: 

:sagittarius: 

:facepunch: 

:1234:

**Pull request merged**